src/studio-header/Header.jsx

```diff
@@ -47,11 +47,13 @@ function Header({
       content: intl.formatMessage(messages['header.links.settings']),
       submenuContent: (
         <>
-          <div className="mb-1"><a rel="noopener" href={`${config.STUDIO_BASE_URL}/settings/details/${courseId}`}>{intl.formatMessage(messages['header.links.scheduleAndDetails'])}</a></div>
-          <div className="mb-1"><a rel="noopener" href={`${config.STUDIO_BASE_URL}/settings/grading/${courseId}`}>{intl.formatMessage(messages['header.links.grading'])}</a></div>
-          <div className="mb-1"><a rel="noopener" href={`${config.STUDIO_BASE_URL}/course_team/${courseId}`}>{intl.formatMessage(messages['header.links.courseTeam'])}</a></div>
-          <div className="mb-1"><a rel="noopener" href={`${config.STUDIO_BASE_URL}/group_configurations/${courseId}`}>{intl.formatMessage(messages['header.links.groupConfigurations'])}</a></div>
-          <div className="mb-1"><a rel="noopener" href={`${config.STUDIO_BASE_URL}/settings/advanced/${courseId}`}>{intl.formatMessage(messages['header.links.advancedSettings'])}</a></div>
+          <div className="mb-1 small"><a rel="noopener" href={`${config.STUDIO_BASE_URL}/settings/details/${courseId}`}>{intl.formatMessage(messages['header.links.scheduleAndDetails'])}</a></div>
+          <div className="mb-1 small"><a rel="noopener" href={`${config.STUDIO_BASE_URL}/settings/grading/${courseId}`}>{intl.formatMessage(messages['header.links.grading'])}</a></div>
+          <div className="mb-1 small"><a rel="noopener" href={`${config.STUDIO_BASE_URL}/course_team/${courseId}`}>{intl.formatMessage(messages['header.links.courseTeam'])}</a></div>
+          <div className="mb-1 small"><a rel="noopener" href={`${config.STUDIO_BASE_URL}/group_configurations/${courseId}`}>{intl.formatMessage(messages['header.links.groupConfigurations'])}</a></div>
+          <div className="mb-1 small"><a rel="noopener" href={`${config.STUDIO_BASE_URL}/settings/advanced/${courseId}`}>{intl.formatMessage(messages['header.links.advancedSettings'])}</a></div>
+          <div className="mb-1 small"><a rel="noopener" href={`${config.STUDIO_BASE_URL}/certificates/${courseId}`}>{intl.formatMessage(messages['header.links.certificates'])}</a></div>
+          <div className="mb-1 small"><a rel="noopener" href={`${config.STUDIO_BASE_URL}/course-runs/${courseId}`}>{intl.formatMessage(messages['header.links.publisher'])}</a></div>
```

Comment:
Is the publisher correct? It used to /publisher/

Reply:
Also seems to me the publisher link is behind a flag. I am not able to see it for every course. @mehaknasir can you confirm if publisher link should be there from marco?

Reply:
I copied this url from a couse on staging it was redirected to this link in studio. will check that again.  and okay let me confirm from macro.

Reply:
removed publisher option from dropdown